src/software/thunderscope/robot_diagnostics/drive_and_dribbler_widget.py

```diff
@@ -54,6 +88,33 @@ def value_change(self, value: float) -> str:
         value_str = "%.2f" % value
         return value_str
 
+    def setup_drive_switch_radio(self, title: str) -> QGroupBox:
+        """Create a radio button widget to switch between per-motor and velocity control modes
+
+        :param title: group box name
+        """
+        group_box = QGroupBox(title)
+        dbox = QVBoxLayout()
+        self.connect_options_group = QButtonGroup()
+        radio_button_names = ["Velocity Control", "Per Motor Control"]
+        self.connect_options_box, self.connect_options = common_widgets.create_radio(
+            radio_button_names, self.connect_options_group
+        )
+        self.use_direct_velocity = self.connect_options[DriveMode.VELOCITY]
+        self.use_per_motor = self.connect_options[DriveMode.MOTOR]
+        self.use_direct_velocity.clicked.connect(self.switch_to_velocity)
+        self.use_per_motor.clicked.connect(self.switch_to_motor)
```

Comment:
Since `self.switch_to_velocity` and `self.switch_to_motor` are short one-line functions, we can get rid of them and use an anonymous lambda function instead:
```suggestion
        self.use_direct_velocity.clicked.connect(lambda: self.toggle_control_mode(True))
        self.use_per_motor.clicked.connect(lambda: self.toggle_control_mode(False))
```